lib/Elastica/Transport/AwsAuthV4.php

```diff
@@ -14,7 +14,7 @@
 
 class AwsAuthV4 extends Guzzle
 {
-    protected function _getGuzzleClient($baseUrl, $persistent = true, Request $request)
+    protected function _getGuzzleClient($baseUrl, $persistent = true, Request $request): Client
```

Comment:
missing `string` and `bool`

Reply:
With that typehints it's incompatible with parent class

Reply:
You should typehint the parent class, no? That class belongs to `Elastica\Transport` namespace

Reply:
Ah yes, missed that. Thank you!